include/zephyr/kernel/thread.h

```diff
@@ -121,7 +121,7 @@ struct _thread_base {
 
 #ifdef CONFIG_SCHED_CPU_MASK
 	/* "May run on" bits for each CPU */
-	uint8_t cpu_mask;
+	uint32_t cpu_mask;
```

Comment:
this is de-aligning the structure 

(SCHED_CPU_MASK does not depend on SMP, but check Kconfig help, it does not make sense to enable it without SMP)